corehq/apps/registration/templates/registration/domain_request.html

```diff
@@ -1,8 +1,23 @@
 {% extends "registration/base.html" %}
 {% load i18n %}
+{% load hq_shared_tags %}
 
 {% block title %}New Project Space{% endblock title %}
 
+{% block js %}{{ block.super }}
+    <script src="{% static 'hqwebapp/js/lib/select2/select2.js' %}"></script>
+{% endblock %}
+
+{% block head %}{{ block.super }}
+    <link href="{% static 'hqwebapp/js/lib/select2/select2.css' %}" rel="stylesheet" />
+{% endblock %}
```

Comment:
Maybe, although that decorator requires access to the request object, which the form won't have.  You could probably come up wiht some clever way to insert it into forms automatically, but I think the best solution here is probably just to merge as-is.  If you agree, feel free to self-merge, I'm signed off.